chainercv/evaluations/eval_detection_coco.py

```diff
@@ -0,0 +1,321 @@
+                [#coco_det_eval_1]_
+            ap/iou=0.50:0.95/area=small/maxDets=100, *numpy.ndarray*, \
+                [#coco_det_eval_1]_
+            ap/iou=0.50:0.95/area=medium/maxDets=100, *numpy.ndarray*, \
+                [#coco_det_eval_1]_
+            ap/iou=0.50:0.95/area=large/maxDets=100, *numpy.ndarray*, \
+                [#coco_det_eval_1]_
+            ar/iou=0.50:0.95/area=all/maxDets=1, *numpy.ndarray*, \
+                [#coco_det_eval_2]_
+            ar/iou=0.50/area=all/maxDets=10, *numpy.ndarray*, \
+                [#coco_det_eval_2]_
+            ar/iou=0.75/area=all/maxDets=100, *numpy.ndarray*, \
+                [#coco_det_eval_2]_
+            ar/iou=0.50:0.95/area=small/maxDets=100, *numpy.ndarray*, \
+                [#coco_det_eval_2]_
+            ar/iou=0.50:0.95/area=medium/maxDets=100, *numpy.ndarray*, \
+                [#coco_det_eval_2]_
+            ar/iou=0.50:0.95/area=large/maxDets=100, *numpy.ndarray*, \
+                [#coco_det_eval_2]_
+            map/iou=0.50:0.95/area=all/maxDets=100, *float*, \
+                [#coco_det_eval_3]_
+            map/iou=0.50/area=all/maxDets=100, *float*, \
+                [#coco_det_eval_3]_
+            map/iou=0.75/area=all/maxDets=100, *float*, \
+                [#coco_det_eval_3]_
+            map/iou=0.50:0.95/area=small/maxDets=100, *float*, \
+                [#coco_det_eval_3]_
+            map/iou=0.50:0.95/area=medium/maxDets=100, *float*, \
+                [#coco_det_eval_3]_
+            map/iou=0.50:0.95/area=large/maxDets=100, *float*, \
+                [#coco_det_eval_3]_
+            ar/iou=0.50:0.95/area=all/maxDets=1, *float*, \
+                [#coco_det_eval_4]_
+            ar/iou=0.50/area=all/maxDets=10, *float*, \
+                [#coco_det_eval_4]_
+            ar/iou=0.75/area=all/maxDets=100, *float*, \
+                [#coco_det_eval_4]_
+            ar/iou=0.50:0.95/area=small/maxDets=100, *float*, \
+                [#coco_det_eval_4]_
+            ar/iou=0.50:0.95/area=medium/maxDets=100, *float*, \
+                [#coco_det_eval_4]_
+            ar/iou=0.50:0.95/area=large/maxDets=100, *float*, \
+                [#coco_det_eval_4]_
+            coco_eval, *pycocotools.cocoeval.COCOeval*, [#coco_det_eval_5]_
+
+
+    .. [#coco_det_eval_1] An array of average precisions. \
+        The :math:`l`-th value corresponds to the average precision \
+        for class :math:`l`. If class :math:`l` does not exist in \
+        either :obj:`pred_labels` or :obj:`gt_labels`, the corresponding \
+        value is set to :obj:`numpy.nan`.
+    .. [#coco_det_eval_2] An array of average recalls. \
+        The :math:`l`-th value corresponds to the average precision \
+        for class :math:`l`. If class :math:`l` does not exist in \
+        either :obj:`pred_labels` or :obj:`gt_labels`, the corresponding \
+        value is set to :obj:`numpy.nan`.
+    .. [#coco_det_eval_3] The average of Average Precisions over classes.
```

Comment:
This line uses `Average Precisions`, where the other lines uses `average precisions/average recalls`